include/envoy/network/listener.h

```diff
@@ -65,10 +65,17 @@ class ListenerConfig {
   /**
    * @return std::chrono::milliseconds the time to wait for all listener filters to complete
    *         operation. If the timeout is reached, the accepted socket is closed without a
-   *         connection being created. 0 specifies a disabled timeout.
+   *         connection being created unless continueOnListenerFiltersTimeout() returns true.
```

Comment:
One random thought, thinking about the use case that @silentdai had, would it make sense to have a timeout in the listener filter config as well to enable override on a per-filter basis? In theory you might want to have different timeouts for TLS vs. some other filter. I think the point is probably moot right now, because folks have shallow (maybe typically 1 deep) filter chains, but just a consideration for future.

Reply:
A listener filter can still do their own timeout, to continue or fail, from their opaque config, such as @silentdai did in #7853. This option applies only to when listener level timeout happens.